src/fileAttachment.js

```diff
@@ -3,6 +3,7 @@ import {arrow4, arrow9, arrow11, jszip, exceljs} from "./dependencies.js";
 import {cdn, requireDefault} from "./require.js";
 import {SQLiteDatabaseClient} from "./sqlite.js";
 import {Workbook} from "./xlsx.js";
+import {inferSchema, coerceRow} from "./table.js";
```

Comment:
Nit: sort imports by path? (VS Code’s “Organize Imports” command can do this for you automatically.)